modules/sentinel-automation-rule/vars.tf

```diff
@@ -48,3 +48,33 @@ variable "expiration" {
   type        = string
   default     = null
 }
+
+variable "default_automation_rules" {
+  description = "A list of default automation rules to be deployed."
+  type        = set(string)
+  default     = []
+}
+
+variable "custom_automation_rules" {
+  description = "A list of custom automation rules to be deployed from the custom-workbooks folder."
+  type        = set(string)
+  default     = []
+}
+
+variable "custom_automation_rule_path" {
+  description = "A path to the custom automation rules."
+  type        = string
+  default     = ""
+}
+
+variable "subscription_id" {
+  description = "The subscription in which the resource group is in."
+  type        = string
+  default     = ""
+}
```

Comment:
We can also do a 
```hcl
data "azurerm_subscription" "current" {
}
```
and refer it using `data.azurerm_subscription.current.subscription_id` to avoid the extra input

Reply:
Thanks, let me incorporate this